label_studio/ui_test/readme.txt

```diff
@@ -0,0 +1,72 @@
+UI Automation for Label-studio
+
+What we have?
+_Automation script
+_testCases.txt file
+_files for import (jpg, png, txt, …)
+
+testCases.txt file contains all needed types(Like: image, font, common formats, audio, html, time series)
+And also it contains appropriate formats for each type (Example: jpg, png for image)
+the symbol ‘+’ means that current format will be tested
+the symbol ‘-‘ means that current format will not be tested
+
+[testCases.txt file]
+=====image=====
++jpg
++png
+-gif
+----------
+=====font=====
++txt
+----------
+=====common formats=====
++csv
+-tsv
+-txt
++json
+
```

Comment:
Audio, HTML, PDF, time series are missed

Reply:
I planned to add them as I will add the corresponding codes for them.